_posts/2023-10-18-sql-training-1.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "Getting started with Trino and SQL"
+author: "Manfred Moser"
+excerpt_separator: <!--more-->
+image: /assets/trino-sql.png
+show_pagenav: false
+---
+
+In our training series [Learning SQL with Trino from the experts]({% post_url
+2023-09-27-training-series %}) Martin Traverso, Dain Sundstrom, David Phillips,
+and myself run through the wide range of SQL support and features of Trino with
```

Comment:
```suggestion
and myself will run through the wide range of SQL support and features of Trino with
```
Hasn't all happened yet! Let's make it clear it's still mostly upcoming.

Reply:
Done